### PR TITLE
Updated properties and url construction in annotation pipeline

### PR DESCRIPTION
**Pull request by angelicaochoa**
Signed-off-by: Angelica Ochoa <aochoa4230@gmail.com>

### CODE REVIEW

File: annotationPipeline/src/main/resources/application.properties.EXAMPLE [hunk 1]
```diff
@@ -2,5 +2,5 @@ spring.batch.job.enabled=false
 genomenexus.hgvs=<HOST>/variant_annotation/hgvs/
 genomenexus.xrefs=<HOST>/xrefs/
 genomenexus.isoform_query_parameter=isoformOverrideSource
-genomenexus.hotspot_parameter=summary
+genomenexus.hotspot_mutassessor_parameter=fields=hotspots,mutation_assessor
```
Comment (inodb) [inline annotationPipeline/src/main/resources/application.properties.EXAMPLE]: this is enrichments parameter right? So maybe call it something like `genomenexus.enrichments_parameters`. Or if you want to specifically just set fields, maybe `genomenexus.enrichments_fields`. And then always include `fields=` in the parameter

**Comment by inodb** [comment]
@angelicaochoa I think it would be good to have some test for this. If you send me a MAF that you usually use for testing (ideally public data) I could give it a shot. It's prolly out of the scope of this PR to add an automated test, but I'd anyway would like to be able to manually check. Ofc don't let that block the merging of the PR if this is urgent

**Comment by angelicaochoa** [comment]
@inodb Can we have a follow up PR for testing? I think Ritika is actively using the annotation pipeline rn. I pointed her to jar for her to use in the meantime with these changes

**Comment by inodb** [comment]
@angelicaochoa ok :+1:

File: annotationPipeline/src/main/resources/application.properties.EXAMPLE (hunk 1 above)
Comment (angelicaochoa) [inline annotationPipeline/src/main/resources/application.properties.EXAMPLE]: 👍 much better, I'll update in a moment

File: annotationPipeline/src/main/resources/application.properties.EXAMPLE [hunk 2]
```diff
@@ -2,5 +2,5 @@ spring.batch.job.enabled=false
 genomenexus.hgvs=<HOST>/variant_annotation/hgvs/
 genomenexus.xrefs=<HOST>/xrefs/
 genomenexus.isoform_query_parameter=isoformOverrideSource
-genomenexus.hotspot_parameter=summary
+genomenexus.enrichment_fields=fields=hotspots,mutation_assessor
```
Comment (sheridancbio) [inline annotationPipeline/src/main/resources/application.properties.EXAMPLE]: the double equals sign looks risky, but I guess it is ok if the properties file parser breaks the key/value at the first equals sign. so the URL below is going to look like .............&fields=hotspots,mutation_assessor
which makes more sense than .........&summary=summary

Comment (inodb) [inline annotationPipeline/src/main/resources/application.properties.EXAMPLE]: @angelicaochoa Sorry, this isn't entirely what I meant. Given that the prop is called `enrichment_fields`, it seems redundant to include `fields=` in the value. Better to update the code from `"&" + enrichmentFields` to `"&fields=" + enrichmentFields`. Not that important tho

Comment (sheridancbio) [inline annotationPipeline/src/main/resources/application.properties.EXAMPLE]: I've made #42 to do the change suggested by @inodb

Comment (inodb) [reply annotationPipeline/src/main/resources/application.properties.EXAMPLE]: Thanks!

**Pull request merged**